BabyPatterns/FeedingsVM+FeedingInProgress.swift

```diff
@@ -91,4 +105,23 @@ extension FeedingsVM {
         }
         return feeding
     }
+
+    // TODO: consider making this a global function or ?
+    // This is the third time copying and pasting this, but first time in this target
+    func communicate(feeding: Feeding, action: Common.FeedingAction) {
+        let info = WatchCommunication(type: feeding.type, side: feeding.side, action: action)
+
+        let jsonEncoder = JSONEncoder()
+        guard let d = try? jsonEncoder.encode(info), WCSession.default.isReachable else {
+            // TODO: show communication error?
+            return
+        }
+
+        WCSession.default.sendMessageData(d, replyHandler: { _ in
+            // no op
+        }) { error in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/c7106bd91c271d949fbb3c9cc499512408cb0c64/BabyPatterns/FeedingsVM+FeedingInProgress.swift#L122"></span>Trailing closure syntax should not be used when passing more than one closure argument.<br />
<code>multiple_closures_with_trailing_closure</code> <code>FeedingsVM+FeedingInProgress.swift:122</code></td>
    </tr>
  </tbody>
</table>